packages/kit/src/ignore.ts

```diff
@@ -1,7 +1,9 @@
 import { existsSync, readFileSync } from 'node:fs'
 import ignore from 'ignore'
 import { join, relative, resolve } from 'pathe'
-import { tryUseNuxt } from './context'
+import { tryUseNuxt, useNuxt } from './context'
+
+const checkIgnoreOutdated = (nuxt = useNuxt()) => !nuxt._ignorePatterns?.join(',').startsWith(nuxt.options.ignore.flatMap(resolveGroupSyntax).join(','))
```

Comment:
I think we need a more performant check as `isIgnored` will be called _lots_ of times - once for every single file that globby or chokidar scans.

Reply:
very good point - made improvements now